omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -11,19 +11,57 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
+import org.openmrs.module.ModuleConstants;
+import org.openmrs.module.ModuleException;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
 
 public class SpaActivator extends BaseModuleActivator {
 
 	private final Log log = LogFactory.getLog(getClass());
+	private final String FRONTENT_DIRECTORY = "frontend";
```

Comment:
Spelling error -- this should be `FRONTEND`

Reply:
@joeldenning this has changed in pull request #3. Do you mind reviewing the pull request?